api/constants/backup.go

```diff
@@ -45,8 +49,11 @@ const (
 )
 
 const (
-	BackupDestTypeOSS types.BackupDestType = "OSS"
-	BackupDestTypeNFS types.BackupDestType = "NFS"
+	BackupDestTypeOSS          types.BackupDestType = "OSS"
+	BackupDestTypeNFS          types.BackupDestType = "NFS"
+	BackupDestTypeCOS          types.BackupDestType = "COS"
+	BackupDestTypeS3           types.BackupDestType = "S3"
+	BackupDestTypeS3Compatible types.BackupDestType = "S3_COMPATIBLE"
```

Comment:
I don't think it's necessary to add S3_COMPATIBLE, just use  S3 if it's using S3 protocol

Reply:
But the parameters they two need is not the same.

Reply:
OK, found it

Reply:
`s3Region` field is required in destination of `S3` type while it's not required in `S3Compatible` type.